### ISSUE TITLE
x/sys/unix: PR_SET_PTRACER_ANY missing for linux_arm conditional compilation

### ISSUE THREAD

**Issue by nvanbenschoten** [golang/go#16682]
The `PR_SET_PTRACER_ANY` system error number is missing from the `// +build arm,linux` version of `zerror_${GOOS}_${GOARCH}.go`, but is available for all other linux builds. This results in the compilation of code like https://github.com/backtrace-labs/go-bcd/blob/master/bcd_sys_linux.go to fail on 32-bit arm architectures.

I haven't been able to find any reason why this flag would be missing, so I'm curious if the correct approach would be to add something like:

```
#ifndef PR_SET_PTRACER_ANY
#define PR_SET_PTRACER_ANY  -0x1
#endif
```

directly after [the other default error codes](https://github.com/golang/sys/blob/7486f8ece05835049d82106d98d73fab64f51667/unix/mkerrors.sh#L141) in the `mkerrors.sh` generation script.

**Comment by tklauser** [comment]
This should be fixed by golang/sys@ea9bcade75cb975a0b9738936568ab388b845617.

**Comment by ianlancetaylor** [comment]
Yes, fixed by https://golang.org/cl/37943.  Thanks for pointing that out.
